.markdown_link_check_config.json

```diff
@@ -10,7 +10,7 @@
       "replacement": "{{BASEURL}}/"
     },
     {
-      "pattern": "^https://github.com/open-telemetry/semantic-conventions/(blob|tree)/[^/]+/docs/",
+      "pattern": "^https://github.com/open-telemetry/semantic-conventions/(blob|tree)/[^v][^/]*/docs/",
       "replacement": "LINK-CHECK-ERROR-USE-LOCAL-PATH-TO-DOC-PAGE-NOT-EXTERNAL-URL/"
```

Comment:
@open-telemetry/docs-maintainers I added an exclusion here for referencing tagged content (`v*`), is this ok?

Reply:
resolving for now so this can be merged, and we can figure this out later